src/main/java/com/murabi/api/MurabiServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.murabi.api;
+
+import javax.servlet.http.*;
+import javax.servlet.*;
+import java.io.*;
+
+public class MurabiServlet extends HttpServlet {
+	public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+		res.setContentType("text/html");// setting the content type
+		PrintWriter pw = res.getWriter();// get the stream to write the data
+
+		// writing html in the stream
+		pw.println("<html><body>");
+		pw.println("Welcome to Murabi Servlet");
+		pw.println("</body></html>");
+
+		pw.close();// closing the stream
```

Comment:
[medium] You don't need to return a `content-type: text/html` for a dummy value. You could just simply leave in only the lines:
```java
PrintWriter pw = res.getWriter();
pw.println("Welcome to Murabi Servlet");
pw.close();
```

[medium] Remove the unnecessary comments.